translations/messages.fr.xlf

```diff
@@ -0,0 +1,3059 @@
+      </trans-unit>
+      <trans-unit id="4691729121764741641" datatype="html">
+        <source>You cannot clone a game. This feature might be implemented later.</source>
+        <target>Clôner une partie n&apos;est pas encore possible. Cette fonctionnalité pour être implémentée dans un futur incertain.</target>
+      </trans-unit>
+      <trans-unit id="6976459996912108024" datatype="html">
+        <source>Game is finished.</source>
+        <target>Partie terminée.</target>
+      </trans-unit>
+      <trans-unit id="7800061171704298797" datatype="html">
+        <source>Human</source>
+        <target>Humain</target>
+      </trans-unit>
+      <trans-unit id="6063984594211340121" datatype="html">
+        <source>Pick the level</source>
+        <target>Choisissez le niveau</target>
+      </trans-unit>
+      <trans-unit id="8800476882871783599" datatype="html">
+        <source>Level <x id="INTERPOLATION" equiv-text="{{ i }}"/></source>
+        <target>Niveau <x id="INTERPOLATION" equiv-text="{{ i }}"/></target>
+      </trans-unit>
+      <trans-unit id="3272612818120648715" datatype="html">
+        <source><x id="INTERPOLATION" equiv-text="{{ compo.scores[0] }}"/> points</source>
+        <target><x id="INTERPOLATION" equiv-text="{{ compo.scores[0] }}"/> points</target>
+      </trans-unit>
+      <trans-unit id="8739046962840362623" datatype="html">
+        <source><x id="INTERPOLATION" equiv-text="{{ winner }}"/> won</source>
+        <target><x id="INTERPOLATION" equiv-text="{{ winner }}"/> a gagné</target>
+      </trans-unit>
+      <trans-unit id="8647687729200262691" datatype="html">
+        <source>Draw</source>
+        <target>Match nul</target>
+      </trans-unit>
+      <trans-unit id="208845294817677439" datatype="html">
+        <source> Restart game </source>
+        <target> Recommencer une partie </target>
+      </trans-unit>
+      <trans-unit id="6980197840325276209" datatype="html">
+        <source> Pass a turn </source>
+        <target> Passer son tour </target>
+      </trans-unit>
+      <trans-unit id="1541946220573817384" datatype="html">
+        <source> Take back move </source>
+        <target> Reprendre un coup </target>
+      </trans-unit>
+      <trans-unit id="1944212987695444934" datatype="html">
+        <source>Turn n°<x id="INTERPOLATION" equiv-text="{{ compo.rules.node.gamePartSlice.turn + 1 }}"/></source>
+        <target>Tour n°<x id="INTERPOLATION" equiv-text="{{ compo.rules.node.gamePartSlice.turn + 1 }}"/></target>
+      </trans-unit>
+      <trans-unit id="5675185658977082941" datatype="html">
+        <source>Player <x id="PH" equiv-text="gameStatus.winner.value + 1"/></source>
+        <target>Joueur <x id="PH" equiv-text="gameStatus.winner.value + 1"/></target>
+      </trans-unit>
+      <trans-unit id="515727987845404926" datatype="html">
+        <source> It is your turn. </source>
+        <target> Ce n&apos;est pas votre tour !</target>
+      </trans-unit>
```

Comment:
**not**

Reply:
non non, "C'est votre tour" ici :D (corrigé)